frontend/packages/dev-console/locales/en/devconsole.json

```
@@ -112,11 +112,11 @@
   "Enter the arguments that will be appended to the command.": "Enter the arguments that will be appended to the command.",
   "Image stream tag": "Image stream tag",
   "Image stream image": "Image stream image",
-  "Docker image repository": "Docker image repository",
+  "Image repository": "Image repository",
```

Missed this, will update to Image registry here as well:

```suggestion
  "Image repository": "Image repository",
```